cmd/hlb/command/run.go

```diff
@@ -243,14 +213,27 @@ func Run(ctx context.Context, cln *client.Client, rc io.ReadCloser, info RunInfo
 
 	ctx = codegen.WithImageResolver(ctx, codegen.NewCachedImageResolver(cln))
 
-	var opts []codegen.CodeGenOption
-	if info.Debugger != nil {
-		opts = append(opts, codegen.WithDebugger(info.Debugger))
+	var (
+		opts         []codegen.CodeGenOption
+		inputSteerer *codegen.InputSteerer
+	)
+	if info.Debug {
+		pr, pw := io.Pipe()
+		r := bufio.NewReader(pr)
+		inputSteerer = codegen.NewInputSteerer(info.Stdin, pw)
+
+		debugger := codegen.NewDebugger(cln, os.Stderr, inputSteerer, r)
+		opts = append(opts, codegen.WithDebugger(debugger))
```

Comment:
I'm a bit concerned about merging this before there's a mechanism for programatically controlling the debugger, since it will prevent updating apps that embed HLB and want to do things like shell-on-error.

Reply:
Sure, we can leave it dangling until everything is ready.